CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Ewon Flexy Thingworx Connector Changelog
 
+## Version 3.3.4
```

Comment:
The commit `RELEASE: v3.3.4` is incomplete. Version number not changed in the `pom.xml` file, `jvmrun` file, or relevant `package.html` file(s).

Previous release commit for example, https://github.com/hms-networks/flexy-thingworx-connector/commit/a13af391966fa67993eec7535240106907b3c48d.

There are multiple `package.html` files with the project version number. Since we recently removed the project version number from all but the root `package.html` in another project, we could also do so here.

Reply:
@it-hms pushed a commit to resolve this while I was actively reviewing.

Reply:
yeah, I realized right after I requested review that I did not `git add ` everything I needed.